data/json/items/comestibles/drink.json

```diff
@@ -1066,21 +1066,21 @@
     "phase": "liquid",
     "flags": [ "EATEN_COLD" ],
     "//": "most store-bought oj is fortified with calcium.",
-    "vitamins": [ [ "vitC", 96 ], [ "calcium", 23 ] ],
+    "vitamins": [ [ "vitC", "83700 μg" ], [ "iron", "320 μg" ], [ "calcium", "348600 μg" ] ],
     "fun": 3
   },
   {
     "type": "COMESTIBLE",
     "id": "oj_fresh",
     "looks_like": "oj",
     "name": { "str_sp": "fresh orange juice" },
-    "weight": "249 g",
+    "weight": "248 g",
     "color": "yellow",
     "spoils_in": "2 days",
     "comestible_type": "DRINK",
     "symbol": "~",
     "quench": 35,
-    "calories": 126,
+    "calories": 111,
```

Comment:
Did you account for fruit pulp in the recipe?

Reply:
I see you did. I’ll try and look at this once I find the power button on my new computer.